umh.docs.umh.app/content/en/docs/features/datainfrastructure/unified-namespace.md

```diff
@@ -1,67 +1,114 @@
 ---
 title: Unified Namespace
 menuTitle: Unified Namespace
-description: Exchange events and messages across all your shopfloor equipment, IT / OT systems such as ERP or MES and microservices.
+description: |
+  Seamlessly connect and communicate across shopfloor equipment, IT/OT systems,
+  and microservices.
 weight: 1000
 edition: community
 aliases:
   - /docs/features/unified-namespace
 ---
 
-The Unified Namespace is an event-driven architecture that allows for seamless communication between nodes in a network. It operates on the principle that all data, regardless of whether there is an immediate consumer, should be published and made available for consumption. This means that any node in the network can work as either a producer or a consumer, depending on the needs of the system at any given time.
+The Unified Namespace is a centralized, standardized, event-driven data
+architecture that enables for seamless integration and communication across
+various devices and systems in an industrial environment. It operates on the
+principle that all data, regardless of whether there is an immediate consumer,
+should be published and made available for consumption. This means that any
+node in the network can work as either a producer or a consumer, depending on
+the needs of the system at any given time.
 
-To use any functionalities of the United Manufacturing Hub, you need to use the Unified Namespace as well. More information can be found in our [Learning Hub on the topic of Unified Namespace](https://learn.umh.app/lesson/introduction-into-it-ot-unified-namespace/).
+This architecture is the foundation of the United Manufacturing Hub, and you
+can read more about it in the [Learning Hub article](https://learn.umh.app/lesson/introduction-into-it-ot-unified-namespace/).
 
 ## When should I use it?
 
-An application consists always out of multiple building blocks. To connect those building blocks, one can either exchange data between them through databases, through service calls (such as [REST](https://learn.umh.app/lesson/introduction-into-it-ot-https-rest/)), or through a message broker.
+The Unified Namespace is the ideal solution for exchanging data between
```

Comment:
Don't say ideal, say something "in our opinion provides the best tradeoff for connecting systems in manufacturing / shopfloor scenarios"